nav2_loopback_sim/nav2_loopback_sim/loopback_simulator.py

```diff
@@ -165,29 +208,35 @@ def timerCallback(self):
 
         self.publishTransforms(self.t_map_to_odom, self.t_odom_to_base_link)
         self.publishOdometry(self.t_odom_to_base_link)
-        self.publishLaserScan()
+        self.mat_map_to_odom = transformStampedToMatrix(self.t_map_to_odom)
+        self.mat_odom_to_base = transformStampedToMatrix(self.t_odom_to_base_link)
```

Comment:
These are non-static, do not store, get when needed. We don't want to have duplicate information we're trying to keep consistent.